__tests__/integration.test.js

```diff
@@ -110,3 +109,49 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200 - Responds with an array of comments from given article id in descending order", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments.length).toBe(11);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+        comments.forEach((comment) => {
+          expect(Object.keys(comment)).toEqual([
+            "comment_id",
+            "body",
+            "article_id",
+            "author",
+            "votes",
+            "created_at",
+          ]);
```

Comment:
Ideally we want to be asserting against the values as well, we know what the article_id will be on each element so we can be specific about that, for the rest its fine to assert against the types.